packages/connect/downloadSpecs.js

```diff
@@ -0,0 +1,46 @@
+import http from 'https';
+import fs from 'fs';
+
+const commonBridgesRepo = 'https://raw.githubusercontent.com/paritytech/smoldot/main/bin';
```

Comment:
Bridges?  I think you forgot which team you're on today :stuck_out_tongue:

Reply:
Got this code from bridges team :) Its an honest mistake